website/docs/r/appstream_image_builder.html.markdown

```diff
@@ -56,8 +56,8 @@ The following arguments are optional:
 
 The `access_endpoint` block supports the following arguments:
 
-* `endpoint_type` - (Required) Type of interface endpoint.
-* `vpce_id` - (Optional) Identifier (ID) of the VPC in which the interface endpoint is used.
+* `endpoint_type` - (Required) Type of interface endpoint. Valid values: `STREAMING`
+* `vpce_id` - (Optional) Identifier (ID) of the interface VPC endpoint.
```

Comment:
No notes here, just _wow_ I had to read the AWS docs, this, and the argument name several times to wrap my head around it because of the weird description issues 😅. Thanks for wording this in a way that made it make sense!